modules/@apostrophecms/admin-bar/index.js

```diff
@@ -13,6 +13,80 @@ module.exports = {
     // Do include a page tree button in the admin bar
     pageTree: true
   },
+  commands(self) {
+    return {
+      add: {
+        [`${self.__meta.name}:undo`]: {
```

Comment:
This is fine, but in a future release it would be great if the lack of a `:` meant it automatically supplied `self.__meta.name` during the command build process.